core/src/main/java/org/p2p/core/utils/AmountExtensions.kt

```diff
@@ -105,10 +105,21 @@ fun BigInteger.isLessThan(value: BigInteger) = this.compareTo(value) == -1
 fun BigInteger.isMoreThan(value: BigInteger) = this.compareTo(value) == 1
 fun BigInteger.isZeroOrLess() = isZero() || isLessThan(BigInteger.ZERO)
 
-fun BigDecimal.asCurrency(currency: String): String =
-    if (lessThenMinValue()) "<${currency}0.01" else "${currency}${formatFiat()}"
+fun BigDecimal.asCurrency(currency: String): String = when {
```

Comment:
asCurrency  -> formatWithPrefixCurrency
asCurrencyAfter -> formatWithPostfixCurrency
currency -> currencySymbol

asCurrency implies that return type would be Currency, not string
also format tells you that it formats stuff
currency can be anything - USD / dollar / cents / $